src/stores/file.js

```diff
@@ -207,6 +209,9 @@ class FileStore {
     const { toast } = this.getInstance();
 
     toast.show(err.message, { style: styles.errorToast });
+    if(err && err.errorLength){
+      highlight(err.errorPosition, err.errorLength);
```

Comment:
this is only working because `onHandleError` is being called in a promise.  There has to be a better way around triggering this action